chore: Release 2213.01 to Insider and Production (#1544)
microsoft/accessibility-insights-windows: Channels/Production/release_notes.md

> ## September 27 2022 Production Release ([v1.1.2089.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.2089.01))

Author: @asksep, this is identical to the Insider notes, other than saying "Production" instead of "Insider"

Welcome to the January 30 2023 Production release of Accessibility Insights for Windows.

Installation Link: https://github.com/microsoft/accessibility-insights-windows/releases/download/v1.1.2213.01/AccessibilityInsights.msi

Documentation Link: https://accessibilityinsights.io/docs/en/windows/overview

### Highlights

- [New Features](#new-features)
- [Rule Updates](#rule-updates)
- [Bug Fixes](#bug-fixes)

#### New Features

> - We have added new features to improve the user experience for users who are blind:

Reviewer: ```suggestion
- We have added new features to improve the experience for blind and vision impaired users of the application:
```

>   - Sound options are now "Always", "Never" or "Automatic" (plays sounds if an active screen reader is detected) [#1495](https://github.com/microsoft/accessibility-insights-windows/pull/1495)

Author: I intentionally listed Automatic last to make it simpler to add the explanation. I don't think it's crucial to try to align the order with the release notes, but I'll defer to @asksep if he wants this reworded

- Display MSAA role names as strings (in addition to the numeric value) to describe `LegacyIAccessiblePattern.Role` values [#1454](https://github.com/microsoft/accessibility-insights-windows/issues/1454)

#### Rule Updates

- Update from [Axe.Windows 1.1.7](https://github.com/microsoft/axe-windows/releases/tag/v1.1.7) to [Axe.Windows 2.1.0](https://github.com/microsoft/axe-windows/releases/tag/v2.1.0)
  - Adjust the `BoundingRectangleSizeReasonable` rule to not fail Telerik Sparkline elements with an area of less than 25 pixels [Axe.Windows #780](https://github.com/microsoft/axe-windows/issues/780)
  - Improve the _How to fix_ text for several rules that previous just repeated the problem description [Axe.Windows #790](https://github.com/microsoft/axe-windows/pull/790), [Axe.Windows #791](https://github.com/microsoft/axe-windows/pull/791)

#### Bug Fixes

- Provide a user-transparent way to request support of unsupport pattern actions [#1420](https://github.com/microsoft/accessibility-insights-windows/issues/1420)
- Target .NET Framework 4.8 to enable new accessibility features [#1465](https://github.com/microsoft/accessibility-insights-windows/issues/1465)

> - Leverage UIA properties (toggle state, items in a list, etc.) to support both braille and user-customized screen readers [#1323](https://github.com/microsoft/accessibility-insights-windows/issues/1323), [#1514](https://github.com/microsoft/accessibility-insights-windows/issues/1514)

Reviewer: "Braille" is a proper name:

``` suggestion
- Leverage UIA properties (toggle state, items in a list, etc.) to support both Braille and user-customized screen readers [#1323](https://github.com/microsoft/accessibility-insights-windows/issues/1323), [#1514](https://github.com/microsoft/accessibility-insights-windows/issues/1514)
```

Author: Fixed
